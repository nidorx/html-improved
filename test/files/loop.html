<vars
    objectVar="{attr1:'One', attr2:'Two'}"
    arrayVar="[{name:'Alex'}, {name:'Gabriel'}, {name:'Darwin'}, {name:'Mara'}]"
    innerVar="[[1,2,3],[4,5,6],[7,8,9,10],[11]]"
    />

<ul>
    <each in="[5,4,3,2,1,0]">
        <li>
            #{$key} - #{$value}
        </li>
    </each>
</ul>

<ul>
    <each in="objectVar">
        <li>
            #{$key} - #{$value}
        </li>
    </each>
</ul>

<ul>
    <each var="value" in="objectVar">
        <li>
            #{$key} - #{value}
        </li>
    </each>
</ul>

<ul>
    <each var="key, value" in="objectVar">
        <li>
            #{key} - #{value}
        </li>
    </each>
</ul>


<ul>
    <each in="arrayVar">
        <li>
            #{$key} - #{$value.name}
        </li>
    </each>
</ul>

<ul>
    <each var="value" in="arrayVar">
        <li>
            #{$key} - #{value.name}
        </li>
    </each>
</ul>

<ul>
    <each var="key, value" in="arrayVar">
        <li>
            #{key} - #{value.name}
        </li>
    </each>
</ul>


<each var="key, value" in="innerVar">
    <each var="subKey, subValue" in="value">
        <p>
            #{key} - #{subKey} - #{subValue}
        </p>
    </each>
</each>
