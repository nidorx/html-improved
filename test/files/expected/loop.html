<ul>
    <li>0-5</li>
    <li>1-4</li>
    <li>2-3</li>
    <li>3-2</li>
    <li>4-1</li>
    <li>5-0</li>
</ul>

<ul>
    <li>
        attr1 - One
    </li>
    <li>
        attr2 - Two
    </li>
</ul>

<ul>
    <li>
        attr1 - One
    </li>
    <li>
        attr2 - Two
    </li>
</ul>

<ul>
    <li>
        attr1 - One
    </li>
    <li>
        attr2 - Two
    </li>
</ul>



<ul>
    <li>
        0 - Alex
    </li>
    <li>
        1 - Gabriel
    </li>
    <li>
        2 - Darwin
    </li>
    <li>
        3 - Mara
    </li>
</ul>

<ul>
    <li>
        0 - Alex
    </li>
    <li>
        1 - Gabriel
    </li>
    <li>
        2 - Darwin
    </li>
    <li>
        3 - Mara
    </li>
</ul>

<ul>
    <li>
        0 - Alex
    </li>
    <li>
        1 - Gabriel
    </li>
    <li>
        2 - Darwin
    </li>
    <li>
        3 - Mara
    </li>
</ul>

<p>0 - 0 - 1</p>
<p>0 - 1 - 2</p>
<p>0 - 2 - 3</p>

<p>1 - 0 - 4</p>
<p>1 - 1 - 5</p>
<p>1 - 2 - 6</p>

<p>2 - 0 - 7</p>
<p>2 - 1 - 8</p>
<p>2 - 2 - 9</p>
<p>2 - 3 - 10</p>

<p>3 - 0 - 11</p>

